### PR TITLE
Bug Fixes: Improved xbutil validate summary report and help

### PR DESCRIPTION
**Pull request by rozumx**
Work done
---------
+ EoU : Added summary of validation tests produced across multiple devices (CR-1092567)
+ EoU : Added quotes around report names in help (CR-1096035)
+ EoU:  Update default xbutil examine reports when using and not using the device option (CR-1096013)
            * If the user doesn't specify one, the default host report will be created.  
            * If the user specifies a device, then the platform and compute-units report will be produced.

### CODE REVIEW

File: src/runtime_src/core/tools/common/XBHelpMenus.cpp [hunk 1]
```diff
@@ -524,17 +524,29 @@ XBUtilities::create_suboption_list_string(const VectorPairStrings &_collection)
...
   unsigned int maxStringLength = 0;
-  for (const auto & pairs : _collection)
+  for (auto & pairs : workingCollection) {
+    // Determine if the keyName needs to have 'quotes', if so add them
+    if (pairs.first.find(' ') != std::string::npos ) {
+      pairs.first.insert(0, 1, '\'');  
+      pairs.first += "\'";     
+    }
+
     maxStringLength = std::max(maxStringLength, (unsigned int) pairs.first.length());
```
Comment (stsoe) [inline src/runtime_src/core/tools/common/XBHelpMenus.cpp:539]: `std::max<unsigned int>(maxStringLength, pairs.first.length())`

File: src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp [hunk 2]
```diff
@@ -1306,6 +1307,154 @@ run_tests_on_devices( xrt_core::device_collection &deviceCollection,
   }
 }
 
+static
+std::string quote_name(const std::string name)
```
Comment (stsoe) [inline src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp:1311]: const std::string&

**Review by stsoe — COMMENTED** [review]
Two comments.

File: src/runtime_src/core/tools/common/XBHelpMenus.cpp (hunk 1 above)
Comment (rozumx) [reply src/runtime_src/core/tools/common/XBHelpMenus.cpp]: Addressed

File: src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp (hunk 2 above)
Comment (rozumx) [reply src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp]: Fixed

**Review by AShivangi — APPROVED** [review]
Thank you for making these changes

**Pull request merged**